WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -46,8 +49,13 @@ public void onErrorResponse(VolleyError volleyError) {
             }
         };
 
+        Map<String, String> params = new HashMap<>();
+        params.put("number", Integer.toString(PeopleUtils.FETCH_USERS_LIMIT));
+        params.put("offset", Integer.toString(offset));
+        params.put("order_by", "display_name");
```

Comment:
I believe this is because WordPress uses `display_name` for the query, but we were attempting to change it to `name` for the API. I would suggest creating a patch to map `name` to `display_name`.